recipes/uriparser/all/conanfile.py

```diff
@@ -25,8 +25,15 @@ class UriparserConan(ConanFile):
         "with_wchar": True,
     }
 
-    _source_subfolder = "source_subfolder"
-    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    @property
```

Comment:
I really don't like converting those to properties. I know that this was done in a lot of recipes already, but it really gives us no advantages (at least I don't see one) and it makes recipes more verbose

Reply:
I agree. I am just following the common trend. I wanted to be hip 😄

Reply:
there is an advantage: read-only. For small recipes, there is no advantage, but for large recipes, it can be useful to assign a wrong variable. in fact, this is not new, I first saw it in the Conan Community.